`go vet` returns `bad syntax for struct tag pair`

It appears there have been a couple of typos in `config.go` and `user.go` when `go vet` is run.

```shell
➜  ip-whitelister git:(main) go version
go version go1.17.13 darwin/amd64
➜  ip-whitelister git:(main) go vet
# github.com/alec-pinson/ip-whitelister
./config.go:26:2: struct field tag `yaml":ttl"` not compatible with reflect.StructTag.Get: bad syntax for struct tag pair
./user.go:28:2: struct field tag `json:"objectId` not compatible with reflect.StructTag.Get: bad syntax for struct tag value
➜  ip-whitelister git:(main)
```

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -3,7 +3,7 @@ name: go test
 on:
   pull_request:
   push:
- 
+
 jobs:
   go:
     runs-on: ubuntu-latest
@@ -16,9 +16,12 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.16
-      - 
+      -
+        name: Vet
+        run: go vet
+      -
         name: Build
         run: go build -v
-      - 
+      -
         name: Test
         run: go test -v

--- a/config.go
+++ b/config.go
@@ -23,7 +23,7 @@ type Configuration struct {
 	Resources   []ResourceConfiguration `yaml:"resources"`
 	Defaults    Defaults                `yaml:"defaults"`
 	IPWhiteList []string                `yaml:"ip_whitelist"`
-	TTL         int                     `yaml":ttl"`
+	TTL         int                     `yaml:"ttl"`
 }
 
 type Defaults struct {

--- a/functions.go
+++ b/functions.go
@@ -22,7 +22,7 @@ func chunkList(array []string, count int) [][]string {
 	lenb := lena/count + 1
 	b := make([][]string, lenb)
 
-	for i, _ := range b {
+	for i := range b {
 		start := i * count
 		end := start + count
 		if end > lena {

--- a/http.go
+++ b/http.go
@@ -75,9 +75,9 @@ var indexTempl = template.Must(template.New("").Parse(`<!DOCTYPE html>
 func (h handle) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	defer func() {
 		if r := recover(); r != nil {
-			// if c.Debug {
-			// 	log.Printf("Handler panic: %v", r)
-			// }
+			if c.Debug {
+				log.Printf("Handler panic: %v", r)
+			}
 		}
 	}()
 	if err := h(w, req); err != nil {

--- a/user.go
+++ b/user.go
@@ -25,7 +25,7 @@ type AzGetGroup struct {
 }
 
 type AzGroup struct {
-	ObjectId string `json:"objectId`
+	ObjectId string `json:"objectId"`
 }
 
 func (u *User) new(client *http.Client, req *http.Request) *User {
@@ -125,9 +125,10 @@ func (u *User) whitelist() {
 	}
 }
 
-func (u *User) unwhitelist() {
-	s := w.delete(u)
-	if s {
-		log.Println("user.unwhitelist(): Whitelisting for '" + u.ip + "' (" + u.name + ") has been removed")
-	}
-}
+// TODO: Currently unused
+// func (u *User) unwhitelist() {
+// 	s := w.delete(u)
+// 	if s {
+// 		log.Println("user.unwhitelist(): Whitelisting for '" + u.ip + "' (" + u.name + ") has been removed")
+// 	}
+// }

